### PR TITLE
Added support for multi-line comments

### PR DESCRIPTION
**Pull request by yogesh19k**
I added a multiline comments map to the comments.go file and made the necessary changes to the countLine function in the utils.go file. I was unable to locate the multiline symbol for some languages, so I did not include them in the map. Please feel free to add any required languages.

### CODE REVIEW

File: utils.go [hunk 1]
```diff
@@ -26,14 +26,37 @@ func countLines(file_name string, ext string) (int32, int32, int32, int32) {
...
 	for {
 		content, _, err := reader.ReadLine()
 		content_str := string(content[:])
 		if err != nil {
 			break
 		}
-		if content_str == "" {
+		//Checks if [Opening symbol] is present at staring of the line
+		if multi_exists && strings.HasPrefix(strings.TrimSpace(content_str),multi_comment_str_open){
+			inside_multi_line_commentt=true
```
Comment (mainak55512) [inline utils.go:47]: "inside_multi_line_commentt" variable is not defined. Please check @yogesh19k. There is a typo...

**Review by mainak55512 — CHANGES_REQUESTED** [review]
Except one typo, the code is working fine and is ready to be merged.
Hi @yogesh19k, please fix the typo.

**Comment by yogesh19k** [comment]
Fixed the typo, Please check

**Review by mainak55512 — APPROVED** [review]
All looks good

**Pull request merged**